runtime/j9vm/jvm.c

```diff
@@ -218,34 +215,36 @@ typedef I_32 (*PortInitLibrary)(J9PortLibrary *portLib, J9PortLibraryVersion *ve
 typedef UDATA (*PortGetSize)(struct J9PortLibraryVersion *version);
 typedef I_32 (*PortGetVersion)(struct J9PortLibrary *portLibrary, struct J9PortLibraryVersion *version);
 
-typedef void* (*J9GetInterface)(J9_INTERFACE_SELECTOR interfaceSelector, J9PortLibrary* portLib, void *userData);
+typedef void *(*J9GetInterface)(J9_INTERFACE_SELECTOR interfaceSelector, J9PortLibrary *portLib, void *userData);
 
 #if defined(J9ZOS390)
-#define ZOS_THR_WEIGHT_NOT_FOUND   ((UDATA) 0U)
-#define ZOS_THR_WEIGHT_MEDIUM      ((UDATA) 1U)
-#define ZOS_THR_WEIGHT_HEAVY       ((UDATA) 2U)
+#define ZOS_THR_WEIGHT_NOT_FOUND ((UDATA)0U)
+#define ZOS_THR_WEIGHT_MEDIUM ((UDATA)1U)
+#define ZOS_THR_WEIGHT_HEAVY ((UDATA)2U)
 
-static UDATA checkZOSThrWeightEnvVar(void);
-static BOOLEAN setZOSThrWeight(void);
+static UDATA
+checkZOSThrWeightEnvVar(void);
+static BOOLEAN
+setZOSThrWeight(void);
```

Comment:
Declarations are normally written on one line (only definitions should be split).